docs/source/api/python/mantid/fixtures.rst

```diff
@@ -0,0 +1,45 @@
+==========================================================================
+ :mod:`mantid.fixtures` --- Pytest Fixtures for projects that use Mantid
+==========================================================================
+
+.. automodule:: mantid.fixtures
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+.. _pytest_fixtures:
+
+Usage
+-----
+To make the fixtures available to your tests, add the following line to your conftest.py file:
```

Comment:
is there a recommended location to insert the statement, such as the very beginning of file `conftest.py`, or right after the `import` statements, or at the very end of the file?

Reply:
I can't find anything that says a recommended location, I would imagine after the imports would be the safest location. That way mantid fixtures would be available if any other fixtures in a `conftest.py` are requesting them. I updated the instruction to put them after the imports.